.travis.yml

```diff
@@ -8,3 +8,6 @@ addons:
   apt:
     packages:
       - libmagick++-dev
+
+bioc_packages:
```

Comment:
That will return an error on Travis because it is not available. See [Travis link](https://travis-ci.org/hadley/pkgdown/builds/242991989?utm_source=github_status&utm_medium=notification)